BLSQ/iaso
# POLIO-1604: Add 'cVDPV1 & cVDPV2' in the Virus type dropdown

Explain what problem this PR is resolving
- Add 'cVDPV1 & cVDPV2' in the Virus type dropdown
Related JIRA tickets : [POLIO-1604](https://bluesquare.atlassian.net/browse/POLIO-1604)

## Self proofreading checklist


## Changes
- Added ("PV1 & cVDPV2", _("PV1 & cVDPV2")) in the virus choices in campaign model
- Added "cVDPV1 & cVDPV2" in the virus constant on the frontend

## How to test
-Create or edit a campaign and check if the new virus type is in the virus dropdown

## Print screen / video
[Screencast from 2024-07-04 15-59-08.webm](https://github.com/BLSQ/iaso/assets/19631540/26e61571-f04a-4877-bfe6-96fd5005c4b9)



[POLIO-1604]: https://bluesquare.atlassian.net/browse/POLIO-1604?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
